### PR TITLE
Install Project Structure

### PR DESCRIPTION
**Pull request by Kartalmiseban01**
Install Project Structure

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
+  "name": "react-redux-todo-app",
+  "version": "1.0.0",
+  "main": "index.js",
+  "repository": "https://github.com/Kartalmiseban01/react-redux-todo-app.git",
+  "author": "Eban Ömer KARTALMIŞ <kartalmiseban@gmail.com>",
+  "license": "MIT",
+  "scripts": {
+    "start": "parcel src/index.html"
```
Comment (mksglu) [inline package.json]: You must define commands for parcel.js bundler to your react project like: `start`, `dev`, `build`
```suggestion
    "start": "parcel src/index.html",
        "start": "cross-env NODE_ENV=development parcel src/index.html --no-cache",
    "build":
      "cross-env NODE_ENV=production parcel build src/index.html --public-url ./"
```
You want to specify the environment, you must install the package:
`yarn add  -D cross-env`

File: src/index.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import {render} from 'react-dom';
+
+const App = () => {
```
Comment (mksglu) [inline src/index.js]: That's component inside should be only `React.DOM`

**Comment by Kartalmiseban01** [comment]
new commit added

**Comment by Kartalmiseban01** [comment]
new commit added;

gitignore 'a dist and .cache are written, package.json' s configuration settings added.

**Pull request closed without merging**